src/aria2.cpp

```diff
@@ -195,6 +197,14 @@ std::string Aria2::tellStatus(const std::string& gid, const std::vector<std::str
   return doRequest(methodCall);
 }
 
+std::string Aria2::makeAriaSecretToken()
+{
+  unsigned seed = std::chrono::system_clock::now().time_since_epoch().count();
+  std::string letters("0123456789abcdefghijklmnopqrstuvqxyzABCDEFGHIJKLMNOPQRSTUVWXYZ");
```

Comment:
For me this does not work, somehow the aria2c operation get stuck in Kiwix-Desktop. After testing around, it seems that the string is too long! If I put `0123456789abcdefghijklmnopqrstuvqxyzA`. It works fine.